### ISSUE TITLE
question about crime categories in the bottom chart

### ISSUE THREAD

**Issue by xiw315** [UBC-MDS/Victorious_Secret_DSCI_532#23]
Hi, there are only two crime categories in the bottom chart while the top chart contains four categories, is that what we intend to present as our final app?

**Comment by aromatic-toast** [comment]
No, as far as I can see, the bottom chart as 4 crime categories as well. Perhaps you need to scroll down to see the Altair drop down?
